maihoang/matching-game/matching-game.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE>
+<html>
+<head>
+    <title> Matching Game </title>
+    <style>
+        div {
+            width: 500px;
+            height: 500px;
+            position: absolute;
+        }
+        #rightSide {
```

Comment:
*rightSide* is id or class name ? 
it uses as an id in html. There is no class name with *rightSide* 
Use camel-case for class